addons/account/models/account_move_line.py

```diff
@@ -1541,6 +1541,25 @@ def copy_data(self, default=None):
                 line._copy_data_extend_business_fields(values)
         return data_list
 
+    def _search_panel_domain_image(self, field_name, domain, set_count=False, limit=False):
+        if field_name != 'account_root_id' or set_count:
+            return super()._search_panel_domain_image(field_name, domain, set_count, limit)
+
+        # Override in order to not read the complete move line table and use the index instead
+        query = self._search(domain, limit=1)
+        query.order = None
```

Comment:
With this, every domain with value `[("id", "in", [])] ` will got the error below. this behavior is intended, isn't it ?

![image](https://user-images.githubusercontent.com/76513233/210706095-bc64fbca-52a1-415f-9295-6c29855218e5.png)

Reply:
No, it is not intended

Reply:
Same issue on Odoo 16.0+e-20230109

Reply:
There is an ongoing fix here https://github.com/odoo/odoo/pull/109271